frontend/packages/console-shared/src/components/getting-started/GettingStartedCard.scss

```diff
@@ -35,13 +35,10 @@
         }
 
         // Override default style for button and a:hover, focus from scaffolding.scss
-        a.pf-c-simple-list__item-link:hover,
-        a.pf-c-simple-list__item-link:focus,
-        a.pf-c-simple-list__item-link:active,
-        button.pf-c-simple-list__item-link:hover,
-        button.pf-c-simple-list__item-link:focus,
-        button.pf-c-simple-list__item-link:active {
-          color: var(--pf-global--Color--100);
+        .pf-c-simple-list {
```

Comment:
I believe this can be removed once https://github.com/openshift/console/pull/9532 lands.